runtime/moonriver/src/governance/councils.rs

```diff
@@ -58,4 +61,7 @@ impl pallet_collective::Config<OpenTechCommitteeInstance> for Runtime {
 	type WeightInfo = moonriver_weights::pallet_collective_open_tech_committee::WeightInfo<Runtime>;
 	type SetMembersOrigin = referenda::GeneralAdminOrRoot;
 	type MaxProposalWeight = MaxProposalWeight;
+	type DisapproveOrigin = EnsureRoot<AccountId>;
+	type KillOrigin = EnsureRoot<AccountId>;
+	type Consideration = ();
```

Comment:
Same as commented above, we need to ask the foundation, which origin they want to use here.

Reply:
@stiiifff for your reference

Reply:
Change to `FastGeneralAdminOrRoot`.